milli/src/heed_codec/facet/mod.rs

```diff
@@ -1,13 +1,9 @@
 mod facet_level_value_f64_codec;
-mod facet_level_value_i64_codec;
 mod facet_value_string_codec;
 mod field_doc_id_facet_f64_codec;
-mod field_doc_id_facet_i64_codec;
 mod field_doc_id_facet_string_codec;
 
 pub use self::facet_level_value_f64_codec::FacetLevelValueF64Codec;
```

Comment:
Maybe we should change this to `FacetLevelValueNumberCodec`

Reply:
As this is a low-level codec implementation that encodes and decodes f64 directly, I prefer keeping the explicit name.